src/components/Forms/Formik/FormikCompactEnvVarSource.tsx

```diff
@@ -82,7 +84,7 @@ export function FormikCompactEnvVarSource({
       name: ""
     });
     setType("Static");
-  }, []);
+  });
```

Comment:
I think this is may be a typo, a useEffect without a pendency array will run on every rerender, which can cause a race condition

Reply:
```
// eslint-disable-next-line react-hooks/exhaustive-deps
}, []);
```